test/index.ts

```diff
@@ -2,6 +2,8 @@ import { expect } from 'chai';
 import { calculateRewardsAmounts, zero, Reward } from '../src/index';
 import { describe } from 'mocha';
 import { BigNumber, ethers } from 'ethers';
+import { mockStakingAmount, totalStakingAmount } from '../src/mocks/mock-data';
+require('dotenv').config();
```

Comment:
Sorry minor nitpick here, but mind using the `import` syntax here instead of `require`?